astroquery/jplhorizons/__init__.py

```diff
@@ -51,6 +51,8 @@ class Conf(_config.ConfigNamespace):
                    'phasecoeff': ('phasecoeff', 'mag/deg'),
                    'solar_presence': ('solar_presence', '---'),
                    'flags': ('flags', '---'),
+                   'R.A._(ICRF)': ('RA', 'deg'),
+                   'DEC_(ICRF)': ('DEC', 'deg'),
                    'R.A._(ICRF/J2000.0)': ('RA', 'deg'),
```

Comment:
both of these are now present, or it's rather a rename of `R.A._(ICRF/J2000.0)` to `R.A._(ICRF)`?

Reply:
I think the latter but there is no official word in the News or Documentation and the underlying telnet interface still mentions `ICRF/J2000.0` as the reference frame. I think it's safe to have both, but will defer to @mommermi judgement

Reply:
I'm fine with keeping both definitions and if it's only to keep the tests alive. There should be no issues with name conflicts.